src/main/java/com/zendesk/maxwell/Maxwell.java

```diff
@@ -73,7 +52,8 @@ private void run(String[] argv) throws Exception {
 		AbstractProducer producer = this.context.getProducer();
 		AbstractBootstrapper bootstrapper = this.context.getBootstrapper();
 
-		final MaxwellReplicator p = new MaxwellReplicator(this.savedSchema, producer, bootstrapper, this.context, this.context.getInitialPosition());
+		MysqlSchemaStore mysqlSchemaStore = new MysqlSchemaStore(this.context);
```

Comment:
does `MysqlSavedSchema.restore` throw `SQLException`? (asking as it would then just log and return, wondering if we want the same behavior with the new code... although I guess it'd need to rescue the run loop, and now it gets triggered by `getSchema`, which can happen in many places I suppose)